### DISCUSSION TITLE
response code 302 on server

### DISCUSSION THREAD

**Discussion by mohamed-alashry** [livewire/livewire#3169] [4. General]
I have component that in the main layout and it working greet. the problem is with the component using different layout than the main one the requests to livewire return response 302  then redirect to login (only on the live server, working locally).

my main problem is that I don't know what is causing this issue. it's just return response code 302.

p.s.
this component doesn't have authentication

**Comment by khalidaminee** [comment]
I have the same problem.

**Comment by khalidaminee** [comment]
Solved: the problem is this that the default route ('/')  in RouteServiceProvider.php - public const HOME = '/'
was made under auth middleware in web.php. remove the auth middleware and leave unauthrized this route. It will work correctly.
